prow/config.yaml

```diff
@@ -283,6 +283,26 @@ presets:
     env:
       - name: KYMA_ENCRYPTION_KEY
         value: projects/kyma-project/locations/global/keyRings/kyma-prow/cryptoKeys/kyma-prow-encryption
+  - labels:
+     preset-kyma-gke-nightly-key: "true"
```

Comment:
I think we don't need presets for this. Simply use the domain (already provided as env) as name.

For example, the name will look like:
```
nightly.build.kyma-project.io.key.encrypted
nightly.build.kyma-project.io.cert.encrypted
```

We generate cert for wildcard domain and only one domain is attached, so it will be enough.